src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0"?>
```

Comment:
This is only a CLI skeleton at this point, with no back-end support. Is this going to be OK to merge to the SONiC community?